nixos/tests/vault-postgresql.nix

```diff
@@ -64,6 +64,6 @@ import ./make-test-python.nix ({ pkgs, ... }:
       machine.wait_for_unit("vault.service")
       machine.wait_for_open_port(8200)
       machine.succeed("vault operator init")
-      machine.succeed("vault status | grep Sealed | grep true")
+      machine.succeed("echo \"$(vault status)\" | grep Sealed | grep true")
```

Comment:
Can we reuse the same method as the `vault` test? https://github.com/NixOS/nixpkgs/commit/cd687af9f4825f82cb7c77456d6ffdf826a44d31